packages/cli/tests/unit/agentInstall/installCommand.spec.ts

```diff
@@ -799,6 +800,28 @@ appmap_dir: tmp/appmap
       sinon.stub(commandRunner, 'run').resolves({ stdout: '', stderr: '' });
     });
 
+    it("tells the user if the target directory doesn't exist", async () => {
+      expect.assertions(5);
+
+      const badDirectory = '/tmp/foo';
```

Comment:
Maybe use something more unique here. That's the goto path for me when creating throwaway directories ;)

Reply:
lol, ok